content-repo/extra-docs/articles/IAM-premium-pack-readme.md

```diff
@@ -183,26 +436,48 @@ These playbooks contain error handling tasks where a user is assigned to review
+
+### AD Account Status 
+The AD Account Status generally represents the current status of the account in Active Directory. In addition to being *Enabled* or *Disabled*, there will be cases where the AD Account Status is also empty - which means that the status has not yet been determined, for example, if a playbook started execution and has not yet determined the new status, resetting the AD Account Status is a measure taken to ensure that the Workday integration performs the wrong operation (activation/deactivation) of the user, as it is already being handled.
+Also, it’s important to note that when a user is terminated (gone through the *IAM - Terminate User* flow) then the AD Account Status will be shown as *Disabled*. However, if the user was hired and had their hire date postponed, the user will simply go through an AD Deactivation flow (rather than full termination flow), and their AD Account Status will be changed to *Pending*, as they are actually pending a new activation, and were not really terminated.
+
 
 ### Fields and Mappers
 
 The mappers that are provided out-of-the-box work with the assumption that you did not add any fields. 
 
 The following provides a general idea of how to include additional information in the user profile indicator, and provision it to your available IAM applications. The [Example](#example) below provides detailed instructions for adding a field to work with the ILM content pack.
 
-1. Add the field to the mappers for the Workday, Okta, Active Directory, and for any other IAM integration configured. 
+1. Add the field to the mappers for Workday, Okta, Active Directory, and for any other IAM integration configured. 
 
-   **Note:** To change the mappers, you will need to duplicate both the incoming and outgoing mappers. 
+   > <i>Note:</i> To change the mappers, you will need to duplicate both the incoming and outgoing mappers. 
 
    Ensure that you are adding the fields to the relevant incident types within each mapper.
 
-   * for the Workday incoming mapper, add the field to the *IAM - Sync User* incident type.
-   * for Okta and Active Directory, add the field to the *UserProfile* incident type in both the incoming and outgoing mappers.
+   * For the Workday incoming mapper, add the field to the *IAM - Sync User* incident type.
+   * For Okta and Active Directory, add the field to the *UserProfile* incident type in both the incoming and outgoing mappers.
    
-        **Note:** As part of the configuration of the Active Directory mapper, you must map a value to the OU (organizational unit) required field. To do this, create a transformer that maps a user attribute of your choice to an OU value.
+        > <i>Note:</i> As part of the configuration of the Active Directory mapper, you must map a value to the OU (organizational unit) required field. To do this, create a transformer that maps a user attribute of your choice to an OU value.
```

Comment:
```suggestion
        <i>Note:</i> As part of the configuration of the Active Directory mapper, you must map a value to the OU (organizational unit) required field. To do this, create a transformer that maps a user attribute of your choice to an OU value.
```

This might be the cause of the failing unit tests. Please remove in all places (saw a few more in the file)

@anara123 FYI

Reply:
What should we use instead of **bold**? <i>italics</i> is not the same

Reply:
This is a regression if fails because of that, we have more areas in the docs with this `>`. It's supposed to generate the following component:
![image](https://user-images.githubusercontent.com/38749041/143230305-30407d3c-5b98-47f1-ab6f-8d22b7cad881.png)
https://xsoar.pan.dev/docs/reference/integrations/okta-iam

Reply:
@reutshal fyi removing the redundant newline at line 1 caused the UT to pass